.pre-commit-config.yaml

```
@@ -1,6 +1,6 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
-exclude: '^(prover_snapshots)|(generated_definitions)|(c_emulator/SoftFloat-3e)'
+exclude: '^(prover_snapshots)|(generated_definitions)|(dependencies/softfloat/berkeley-softfloat-3)'
```

I think we shouldn't run the checks on any dependencies? How about this:
```suggestion
exclude: '^(prover_snapshots)|(generated_definitions)|(dependencies)'
```

---

This is still open

---

Oops, fixed.

---

Oops sorry I forgot. It would still be nice to run pre-commit on the CMakeLists.txt file I added there, but I think something like this will work

```
exclude: '^(prover_snapshots)|(generated_definitions)|(dependencies/[^/]+/)'
```

---

Current version is good too, didn't not think about the cmakelists